mantidimaging/gui/windows/main/view.py

```diff
@@ -137,7 +137,9 @@ def __init__(self, open_dialogs: bool = True):
 
         args = CommandLineArguments()
         if args.path():
-            self.presenter.load_stacks_from_folder(args.path())
+            paths_list: List[str] = list(filter(None, args.path()))
+            for filepath in paths_list:
+                self.presenter.load_stacks_from_folder(filepath)
```

Comment:
Not sure why the filter is needed here.

Reply:
Having looked into this more, I agree and have removed